### PR TITLE
CVE-2021-25749: runAsNonRoot logic bypass for Windows containers

### PR DESCRIPTION
**Pull request by ritazh**
Populating CVE details for CVE-2021-25749

### CODE REVIEW

File: 2021/25xxx/CVE-2021-25749.json [hunk 1]
```diff
@@ -2,17 +2,112 @@
...
+                      },
+                      {
+                        "version_name": "kubelet v1.23.0 - v1.23.10",
+                        "version_affected": "<",
+                        "version_value": "kubelet v1.23.11"
+                      },
+                      {
+                        "version_name": "kubelet v1.24.0 - v1.24.4",
+                        "version_affected": "<",
+                        "version_value": "kubelet v1.25.0"
```
Comment (SaranBalaji90) [inline 2021/25xxx/CVE-2021-25749.json:45]: Is this right or should it be 1.24.5?

Comment (ritazh) [reply 2021/25xxx/CVE-2021-25749.json:45]: Good catch. I think it's an error in the original email:
https://groups.google.com/g/kubernetes-security-announce/c/qqTZgulISzA
> Fixed Versions
kubelet v1.22.14
kubelet v1.23.11
kubelet v1.23.5
kubelet v1.25.0

Comment (ritazh) [reply 2021/25xxx/CVE-2021-25749.json]: done

File: 2021/25xxx/CVE-2021-25749.json [hunk 2]
```diff
@@ -2,17 +2,112 @@
...
+            "product": {
+              "product_data": [
+                {
+                  "product_name": "Kubernetes",
+                  "version": {
+                    "version_data": [
+                      {
+                        "version_name": "kubelet v1.22.0 - v1.22.13",
+                        "version_affected": "<",
+                        "version_value": "kubelet v1.22.14"
```
Comment (enj) [inline 2021/25xxx/CVE-2021-25749.json:35]: Do we normally include the component and the version together?

Comment (ritazh) [reply 2021/25xxx/CVE-2021-25749.json:35]: Good question. Looking at other CVEs, https://github.com/kubernetes-security/cvelist-public/blob/883471418667723df545686aa6734ec0e2ec8031/2022/3xxx/CVE-2022-3685.json#L47-L49 is the only one that has the component in the `version_value`. 
Others only have it as part of `version_name`.
https://github.com/kubernetes-security/cvelist-public/blob/883471418667723df545686aa6734ec0e2ec8031/2020/7xxx/CVE-2020-7807.json#L21-L22
https://github.com/kubernetes-security/cvelist-public/blob/883471418667723df545686aa6734ec0e2ec8031/2021/3xxx/CVE-2021-3860.json#L19-L20

We can leave kubelet in `version_name` but remove it from `versnion_value` wdyt?

Comment (ritazh) [reply 2021/25xxx/CVE-2021-25749.json]: @enj updated. PTAL

**Pull request merged**